server/meshmodel/argo-cd/relationships/hierarchical_parent.json

```diff
@@ -0,0 +1,45 @@
+{
+  "schemaVersion": "core.meshery.io/v1alpha2",
+  "version": "v1.0.0",
+  "metadata": {
+    "description": "A parent-child relationship implies the requirement of the parent component before the child component can be created. For example, a Namespace in Kubernetes can be a parent of Pods within that namespace. Similarly, in Argo CD, an AppProject represents a logical grouping of Applications. Applications and ApplicationSets reference their AppProject by name."
```

Comment:
Be sure to capitalize consistently, if you would: Namespace.

Yes, the description is ✅